### PR TITLE
Use UUID native functions

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Use the UUID native functions to generate UUIDs.

#### Ticket Link
Fix https://mattermost.atlassian.net/browse/MM-58743

#### Release Note
```release-note
Improve how we generate UUIDs
```

### CODE REVIEW

**Review by esarafianou — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -97,7 +97,8 @@
     "react-syntax-highlighter": "15.5.0",
     "semver": "7.6.2",
     "tinycolor2": "1.6.0",
-    "url-parse": "1.5.10"
+    "url-parse": "1.5.10",
+    "expo-crypto": "13.0.2"
```
Comment (enahum) [inline package.json:101]: not that it matters much.. but it would be good to sort this

**Review by enahum — CHANGES_REQUESTED** [review]
Podfile.lock needs to be updated

**Comment by larkox** [comment]
@enahum Podfile.lock is updated. Am I missing anything there?

**Review by enahum — APPROVED** [review]
My bad, missed it

**Pull request merged**